docs/subnets/how-to-upgrade-precompile.md

```diff
@@ -27,7 +27,7 @@ It assumes you have already created and deployed a Subnet called `testSubnet`.
 ## Generate the Upgrade File
 
 The 
-[Precompiles](./customize-a-subnet.md#network-upgrades-enabledisable-precompiles) 
```

Comment:
For changes to the directory which will affect URLs, you need to include each change as a line in the `_redirects` file so that any external sites already referencing this page gets redirected to the new URL.